service/src/test/java/bio/terra/user/app/controller/ProfileApiControllerTest.java

```diff
@@ -71,10 +75,44 @@ void multiUser() throws Exception {
     assertUserProfile("$.value.name", "John");
   }
 
+  @Test
```

Comment:
opt nit tests are easier to read with this style naming. If you're testing one method, test name starts with the method that you're testing.

```
getUserProfile_emptyProfile
setUserProfile_requestEmailNotSet
getUserProfile_noValue
multiUser
getUserProfile_requestEmailSet_requesterNotAdmin_throws403
requestEmailSet_requesterIsAdmin


```